src/style.css

```
@@ -0,0 +1,36 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars_star {
+  height: 16px;
+  width: 16px;
+  margin: auto 2px;
+  background-repeat: no-repeat;
+  background-image: url(images/star.svg);
+  background-position: 0;
+}
+
+.stars--1 .stars_star:first-child {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--2 .stars_star:nth-child(-n+2) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--3 .stars_star:nth-child(-n+3) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--4 .stars_star:nth-child(-n+4) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--5 .stars_star:nth-child(-n+5) {
+  background-image: url(images/star-active.svg);
+}
```

use comma comma to avoid code duplicate
```suggestion
.stars--1 .stars_star:first-child,
.stars--2 .stars_star:nth-child(-n+2),
.stars--3 .stars_star:nth-child(-n+3),
.stars--4 .stars_star:nth-child(-n+4),
.stars--5 .stars_star:nth-child(-n+5) {
  background-image: url(images/star-active.svg);
}
```